### ISSUE TITLE
WC Vendors: Custom Tab on product with Seller Info

### ISSUE THREAD

**Issue by AimeeHapp** [AimeeHapp/live#13]
According to WC Vendor documentation (search on Seller Info - Custom Tab) here:

https://www.wcvendors.com/kb/wcvendors-setup-guide/

when you enter Seller Info in the vendors profile, it creates a custom tab on all products that are assigned that vendor called Seller Info

On this product:

https://dev.happivize.com/product/success-and-spirit-2/

it does not have the info, although the vendor Mia Den Haan has seller info on her profile (go to Users>Vendors>Mia den Haan

Can you figure out why WC Vendors is not automatically creating the custom tab? Does it not work on grouped products?

Ideally I would like WC Vendors to auto create the custom tab 

1) ONLY IF the info is completed in Seller Info for the vendor - if it is blank then no tab is created
2) The name of the tab instead of Seller Info would be About <Vendor Name> so in this case About Mia den Haan, whatever name we have listed in the Name Column in Users menu

![image](https://user-images.githubusercontent.com/28787692/34743465-d61d2aa6-f557-11e7-9ffc-96f75afc139b.png)

3) It would display the contents from Seller Info and the image we add using Add Media we can add css easily to make it float left with the padding, like the other one you did today.

Then this way when people have multiple products, we create their about info in one place and it will auto update on all products every time we make a change.

**Comment by happivize** [comment]
One more piece of information, we use a plugin WooCommerce Tab Manager, so I'm guessing it may have something to do with that.

**Comment by shpp-shablyuk** [comment]
Check please.

**Comment by happivize** [comment]
Yes, this looks good, I was able to add class="float-image-left" to the seller info image and it applied. Please include how to add to live

**Comment by shpp-shablyuk** [comment]
https://prnt.sc/hz7mhb

**Comment by happivize** [comment]
Thanks, was looking at it on another product of Mia's, and it's populating
her name on the tab but it is blank. It's the same on live. Can you tell
why?

https://dev.happivize.com/product/divine-soul-essence-mia-den-haan/#tab-seller_info

**Comment by shpp-shablyuk** [comment]
https://dev.happivize.com/wp-admin/post.php?post=40512&action=edit

You should add closing tag "</em>"
https://prnt.sc/hzgy64

**Comment by happivize** [comment]
Thanks, good to know, I would never have thought another tab would affect
that. It was also turning my top admin bar italicized, I wondered why that
was happening. After I took that first tag out (I didn't want it em
anyway), toolbar went back to normal and tab worked properly.

**Comment by happivize** [comment]
I'm trying to insert an image here, and even though I select thumbnail, it displays it as super large.

https://dev.happivize.com/product/energy-management-essentials-special-offer-chameice-daniel/

user is in Vendor, Chameice Daniel.

**Comment by happivize** [comment]
Also I tried to insert an image into Elementor and same thing is happening, even if I choose another size it is putting it in as the full size. I don't know if it is related but thought I would mention

**Comment by shpp-shablyuk** [comment]
The shortcode "image" inserts the CSS class "size-thumbnail". But for this class there are no CSS styles. Because of this, the image is inserted in full size. I added a style 
`
.size-thumbnail {
    max-width: 220px;
}
`

**Comment by happivize** [comment]
Tested and that works, how do I fix on Live?

**Comment by shpp-shablyuk** [comment]
The simplest solution is to make changes to the plugin. 
https://prnt.sc/i24xdm
To do this, you must add two slash at the beginning of line 70 at the specified location. But in this case, when you update the plugin, these changes will be deleted.
It suits you?

**Comment by happivize** [comment]
I forgot to mention that I tried to do this, but it caused a fatal error
and Wordpress wouldn't accept it. I did it by going to Plugin Editor and
pulling up that plugin and going to line 70 and confirmed it was same ine
as you suggested.

**Comment by shpp-shablyuk** [comment]
I couldn't make changes on the dev site using the wordpress editor. We should to make changes via FTP access or by using the hosting file manager

**Comment by happivize** [comment]
Thanks, doing it via FTP worked. Closing issue.

**Comment by happivize** [comment]
The About Tab is showing the user name not the Name. Please fix to show Name so that it has spaces. Please see video.

https://www.screencast.com/t/sVG8fZ2A3X

**Comment by shpp-shablyuk** [comment]
Fixed.
Check please.
https://dev.happivize.com/product/virginia/

**Comment by happivize** [comment]
thanks please provide code for live

**Comment by shpp-shablyuk** [comment]
You should update the snippet https://prnt.sc/idt5ub

**Comment by happivize** [comment]
I'm glad when you say it's snippets!

**Comment by happivize** [comment]
has been working for awhile

**Comment by AimeeHapp** [comment]
Hey Sergey, can you fix this...must have broken with an update. The photo is no longer displaying on the Vendor tab like it is supposed to, just showing the code at the top.

![image](https://user-images.githubusercontent.com/28787692/61592432-24faa080-aba1-11e9-8c5f-37a73e2c76c2.png)

Please let me know.

Thanks,
Aimee

**Comment by shpp-shablyuk** [comment]
Hi Aimee,
My credential for wp dashboard doesn't work.
